packages/sdk-react/src/components/payables/PayableDetails/PayableDetailsForm/helpers.ts

```diff
@@ -225,3 +226,30 @@ function formatTaxToMinorUnits(tax: number): number {
 function formatTaxFromMinorUnits(tax: number): number {
   return tax / 100;
 }
+
+export const isFieldRequired = (
+  fieldName: string,
+  ocrRequiredFields: Record<string, boolean> | undefined,
+  value?: string | null
+) => {
+  const defaultRequiredFields: Record<string, boolean> = {
+    invoiceNumber: true,
```

Comment:
I would really recommend naming the fields in the same way as they are specified in OpenAPI. Otherwise, it would be painful to document.

Reply:
@radist2s Alex, probably, we should create a pseudo component MoniteThemeSettings that will allow us to pass settings like ocrFields and date format?